tests/translator/input/graphqlapi_resolver_ref_function_by_logical_and_function_id.yaml

```diff
@@ -28,7 +28,6 @@ Resources:
           saveNote:
             InlineCode: blahblahblah
             FieldName: my_field
-            DataSource: NONE
```

Comment:
Why was this removed?

Reply:
ah, no DataSources in Resolver. They where there to be applied to all Functions in the resolver, but now we have `Defaults`